.changeset/brown-dingos-switch.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/tree": minor
+"__section": feature
+---
```

Comment:
Users of tree if they are using our public API surface should import it from the reexports in fluid-framework so pretty much all tree changes (except those only impacting legacy APIs) should be in both. Also use the tree section (I'm not super clear why we have that section, but it should be used when making tree changes)

```suggestion
---
"@fluidframework/tree": minor
"fluid-framework": minor
"__section": tree
---
```